proto/buf/alpha/registry/webhook/v1alpha1/event_payloads.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.webhook.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/repository.proto";
+import "buf/alpha/registry/v1alpha1/repository_commit.proto";
+import "buf/alpha/registry/v1alpha1/webhook.proto";
+import "google/protobuf/timestamp.proto";
+
+// EventRequest is the request payload that will be sent to the customer that is
+// subscribed to webhook events in the BSR. This request message serves both as
+// (1) documentation of the payload that will be sent in a JSON blob, or (2)
+// using this in an rpc signature in your own proto file.
+message EventRequest {
+  // The event that was triggered.
+  buf.alpha.registry.v1alpha1.WebhookEvent event = 1;
```

Comment:
Why is this not in the `registry.webhook.v1alpha1` package, ie why is `WebhookEvent` in a different package than `EventRequest`?

Reply:
The event types were defined for internal usage [when registering a webhook tied to an event](https://github.com/bufbuild/buf/blob/5211c6467af3ee6c01b3dc20e2c13e8b415c0f3e/proto/buf/alpha/registry/v1alpha1/webhook.proto#L36).

This PR originally included the external RPC that would be implemented by customers, so I got [the suggestion](https://github.com/bufbuild/buf/pull/1249#discussion_r911236908) to change it to another package because this is something for the customer to implement, not something that belongs to the BSR. Since still it's something related to the BSR usage, I proposed making it a `registry` subpackage.

But now, since we're not defining any `rpc` here, I don't see it wrong to make it live within the original `registry` package. @robbertvanginkel wdyt?

Reply:
Following those comments, I don't think they explain why `WebhookEvent` (which is not a request or response for anything in `registry`) would be in a different package than `EventRequest`.